src/routes/components/collapsible/+page.md

```diff
@@ -0,0 +1,125 @@
+<script lang="ts">
+    import Collapsible from "$lib/components/Collapsible.svelte";
+    let toggleButton: HTMLElement;
+    let toggleContentF: function;
+    let expanded: boolean;
+    let message: string;
+    $: message = expanded ? 'close' : 'open';
+</script>
+
+# Collapsible
+
+Collapsible is an accordion element that expands when clicked on. They allow you to hide content that is not immediately relevant to the user. Buttons or anchors are used as triggers that are mapped to specific elements you toggle.
+
+# Usage
+
+```html
+<Collapsible iconSize="medium">
+  <div slot="header">About smart contracts</div>
+  <p>
+    Advanced smart contracts process HTTP requests, control other chains, and
+    scale infinitely.
+  </p>
+</Collapsible>
+```
+
+#### With an external button
+
+```html
+<script>
```

Comment:
noticed few things so here a suggestion of example, let me know what you think?

```
<script lang="ts">
  let expanded = false;
</script>

<button class="secondary"
  on:click={() => expanded = !expanded}
>
  Press to open
</button>

<Collapsible iconSize="medium" 
  expandButton={false} externalToggle={true} bind:expanded>
  <div slot="header" class="header_div" id="header2">
    Collapsible with external button
  </div>
  <div class="content_text">
    This collapsible uses an external button element to toggle open its content.
  </div>
</Collapsible>
```

Reply:
This is much simpler, thank you!